sdk/monitor/azure-monitor-opentelemetry-exporter/azure/monitor/opentelemetry/exporter/_quickpulse/_live_metrics.py

```diff
@@ -60,6 +63,7 @@ def enable_live_metrics(**kwargs: Any) -> None:
     :rtype: None
     """
     _QuickpulseManager(kwargs.get('connection_string'), kwargs.get('resource'))
+    set_statsbeat_live_metrics_feature_set()
```

Comment:
Would this be set whenever it's "enabled" or specifically turned on? Also, NIT, isn't there already a way to tell when it's enabled/on via the manager?